oc_dms_mirror/dms_mirror.py

```
@@ -137,15 +137,16 @@ def process_component_webhook(self, payload):
         :param str payload: Webhook payload
         :return: 'None' on success, raised exception on failure
         """
-        component = payload.get('componentVersion').get('component')
-        version = payload.get('componentVersion').get('version')
         event_type = payload.get('type')
 
         if event_type != self.DmsEventType.PUBLISH_COMPONENT_VERSION.value:
-            _err_msg = f"Skipping {component} since event type not 'PUBLISH_COMPONENT_VERSION'"
+            _err_msg = f"Event type is {event_type}, skipping"
             self.logger.error(self.__log_msg(_err_msg))
             raise Exception(_err_msg)
```

Ruff (0.14.2)

Create your own exception

(TRY002)